README.md

```diff
@@ -1,6 +1,24 @@
+![Screenshot](https://github.com/EppMouse/Eventual/raw/master/screen.PNG)
+
 Eventual
 =================
 Personal time management system for Haiku. Intended to be a full-featured organizer.
 
+Features:
+
+*Event name
+

```

Comment:
Remove the empty new lines between bullet points and ad a space after the "*" to have it correctly formatted for markdown.